app/2.0/start/first-element/intro.md

```diff
@@ -95,30 +96,17 @@ components.
     demo/
     icon-toggle-finished/
     icon-toggle.html
+    index.html
     </pre>
 
 2.  Run the Polymer development server from the project directory:
 
-        polymer serve
-
-3.  `polymer serve` gives you two URLs to choose from - one for an application, another for 
-reusable components. Open the URL for **reusable components**. 
-
-    <img src="/images/2.0/first-element/polymer-serve-urls.png" alt="Output from the polymer serve 
-command, showing URLs" title="Local server URLs">
-
-    (Note that the path uses `icon-toggle`—the
-    component name listed in this element’s `bower.json` file—rather than the actual directory name.
-    If you’re wondering why `polymer serve` does this, see [HTML imports and dependency
-    management](/2.0/docs/tools/polymer-cli#element-project-layout).)
+        polymer serve --open
 
     You’ll see some text where the icon toggles should appear. It doesn’t look
     very interesting, but it shows that everything is working.
-
-**If this text doesn’t appear**, make sure you’re looking at the the demo folder itself, or at 
-`demo/index.html`,
-not at `toggle-icon.html` or `demo-element.html`.
-{ .alert .alert-info }
+ 
+    (Note that the path uses `icon-toggle`—the component name listed in this element’s `bower.json` file—rather than the actual directory name. If you’re wondering why `polymer serve` does this, see [HTML imports and dependency management](/2.0/docs/tools/polymer-cli#element-project-layout).)
```

Comment:
Since we don't show the path explicitly anymore—maybe instead of "the path uses", say, "the URL includes"